frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -120,7 +114,7 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 )
               }
               {
-                isParticipant(currentUser) && (
+                !hasRole(currentUser, "ROLE_RIDER") && isParticipant(currentUser) && (
```

Comment:
same here...

Reply:
Same here again... you marked it as resolved but didn't make the change?

Reply:
Sorry professor. The code we wrote earlier did not work and I was failing a lot of the tests because of it. I realized it is because of the fact that we have an AND statement but we need an OR between the two cases. I changed the code and submitted a new pr!